mesonbuild/dependencies/misc.py

```diff
@@ -6,6 +6,7 @@
 
```

Comment:
I think there is a typo in the commit message which makes it hard to understand. I think "for omp_lib.h" should be "or omp_lib.h".

Reply:
Agreed, it was hard to understand. I've reworded it